services/identity/__tests__/AuthService.spec.ts

```diff
@@ -33,39 +28,19 @@ describe("Auth Service", () => {
 
     // Assert
     expect(actual).toBe(expected)
-    expect(mockAxios.get).toHaveBeenCalledWith(mockEndpoint, mockHeaders)
+    expect(mockGitHubService.checkHasAccess).toHaveBeenCalledWith(
+      { accessToken: mockAccessToken, siteName: mockSiteName },
+      { userId: mockUserId }
```

Comment:
wdyt about creating a mock variable and storing these params there to avoid repetition?

Reply:
Good idea, 3e6c052